hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationSupervisorMetrics.java

```diff
@@ -66,6 +66,9 @@ public void getMetrics(MetricsCollector collector, boolean all) {
             supervisor.getQueueSize())
         .addGauge(Interns.info("numRequestedReplications",
             "Number of requested replications"),
-            supervisor.getReplicationRequestCount());
+            supervisor.getReplicationRequestCount())
+        .addGauge(Interns.info("numTimeoutReplications",
+            "Number of timeout replications in queue"),
```

Comment:
Possible clarification:

```suggestion
            "Number of replication requests timed out before being processed"),
```